src/LinLogQuantization.jl

```diff
@@ -2,6 +2,8 @@ module LinLogQuantization
 
     export  LinQuantArray, LogQuantArray,
     LinQuant8Array, LinQuant16Array, LinQuant24Array, LinQuant32Array,
+    LinQuantUInt8Array, LinQuantUInt16Array, LinQuantUInt24Array, LinQuantUInt32Array,
+    LinQuantInt8Array, LinQuantInt16Array, LinQuantInt24Array, LinQuantInt32Array,
```

Comment:
I think if we avoid defining new types then we can make this pull request non-breaking. So a `LinQuant8Array` of type `T <: Signed` would be what you're suggesting and `T <: Unsigned` is what we had before.